src/styles/index.css

```diff
@@ -0,0 +1,104 @@
+@import "./stars.css";
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  --color-card-main: #060b35;
+  --color-card-add: #616070;
+  --color-link: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  margin: 32px auto 40px;
+  height: 134px;
+}
+
+.card__picture img {
+  height: 100%;
+  width: auto;
+  object-position: center;
+}
+
+.card__info {
+  margin: 0 16px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--color-card-main);
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-card-add);
+  margin: 0 auto 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-card-main);
+}
+
+.card__price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
```

Comment:
It's better to use a percentage or multiplier value for line height.
`line-height: 1.4`, `line-height: 150%`, etc.